packages/node-resolve/src/resolveImportSpecifiers.js

```diff
@@ -11,8 +11,11 @@ import { isDirCached, isFileCached, readCachedFile } from './cache';
 const resolveImportPath = promisify(resolve);
 const readFile = promisify(fs.readFile);
 
-const pathNotFoundError = (subPath, pkgPath) =>
-  new Error(`Package subpath '${subPath}' is not defined by "exports" in ${pkgPath}`);
+const pathNotFoundError = (importer, subPath, pkgPath) =>
+  new Error(
+    `Could not resolve import in ${importer}.` +
+      ` Package subpath '${subPath}' is not defined by "exports" in ${pkgPath}`
```

Comment:
I would usually add the importer second not first, like a sort of stack, but this is mostly just personal preference really.

Reply:
Can we also add the import as written in the importer? And possibly JSON.stringify or quote the ids? I.e.

```
Could not resolve import "foo/bar" in "/path/to/importer". Package sub-path "/bar" is not defined by "exports" in "/path/to/foo/package.json".
```

But this is close to bike-shedding, the improvement is definitely helpful as it is.

Reply:
Good idea, I've added this as well.